Themis/Models/Repository.swift

```diff
@@ -28,6 +28,9 @@ class Node: Hashable, ObservableObject {
     let type: FileType
     var children: [Node]?
     @Published var code: String?
+    var templateCode: String?
+    @Published var diffAdded: [Int]? // line numbers of changed lines starting from 1
+    @Published var diffRemoved: [Int]?
```

Comment:
Maybe include in the variable name that these are lines.
```suggestion
    @Published var diffLinesAdded: [Int]? // line numbers of changed lines starting from 1
    @Published var diffLinesRemoved: [Int]?
```